ReactiveSwift.podspec

```diff
@@ -21,5 +21,5 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = {"OTHER_SWIFT_FLAGS[config=Release]" => "$(inherited) -suppress-warnings" }
 
   s.cocoapods_version = ">= 1.4.0"
```

Comment:
`swift_versions` DSL is only available from CocoaPods 1.7.0, so this line should be updated to `">= 1.7.0"` as well.